DigitalLearningSolutions.Web/Views/ResetPassword/Error.cshtml

```diff
@@ -0,0 +1,9 @@
+@using DigitalLearningSolutions.Web.ViewModels.Common
+<h1 class="nhsuk-heading-xl">Something went wrong...</h1>

```

Comment:
I'm not 100% sure about the "..." at the end of the title. I think it's worth taking a screenshot of the page and posting in the HEE external to see if they have any comments about the content of this page :)